### ISSUE TITLE
Should we still default to a bundled taglib?

### ISSUE THREAD

**Issue by subnut** [clangen/musikcube#510]
This is not true anymore -
https://github.com/clangen/musikcube/blob/e4593ba0db880b57febbbc9a87e0c4e7bd9a8a70/.cmake/ConfigureTaglib.cmake#L1-L3

Taglib recently had a release on 2021 -
https://github.com/taglib/taglib/releases/tag/v1.12

**Comment by clangen** [comment]
Yeah, I'm going to be re-evaluating this sometime in the near future (within the next couple releases). While it's true there was a recent release with a lot of bugfixes, I believe it's still not widely available via OS-provided packages in most of the Linux distros we support. :(

**Comment by subnut** [comment]
### Taglib status in all distros (this image is updated automatically)
[![Packaging status from Repology](https://repology.org/badge/vertical-allrepos/taglib.svg)](https://repology.org/project/taglib/versions)
